react/features/base/conference/reducer.js

```diff
@@ -183,19 +183,31 @@ function _conferenceJoined(state, { conference }) {
  * reduction of the specified action.
  */
 function _conferenceLeft(state, { conference }) {
-    if (state.conference !== conference) {
-        return state;
+    let output = state;
```

Comment:
I've seen that referred to as `nextState` when reading redux-involved source code so I'll change `output` which to me is more generic to `nextState` which to me is more specific.